### PR TITLE
Wfluxor

### PR DESCRIPTION
**Pull request by vmcj**
I thinks those are the changes which are needed during the WF, I'll setup some VMs on Calcb to let Nicky play with it.

This is not meant to be merged, but makes it easier to discuss possible changes.

### CODE REVIEW

**Comment by vmcj** [comment]
PHPStorm fails (@nickygerritsen you always fix this?)
Emergency group is wrong for clusterssh
Keepalive prio is not set.

**Comment by nickygerritsen** [comment]
> PHPStorm fails (@nickygerritsen you always fix this?) Emergency group is wrong for clusterssh Keepalive prio is not set. Not replacing invalid character(s) "{'-'}" in group name (wf46-emergency)

Phpstorm you indeed need to manually copy. The readme contains information on how to do it. Hopefully someone else can do it this year while I’m not there yet

File: provision-contest/ansible/domserver.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@
         use_ssh_args: true
       notify: Clear application cache
 
-    - name: Grant www-data permisions to image folders
+    - name: Grant www-data permissions to image folders
```
Comment (vmcj) [inline provision-contest/ansible/domserver.yml:86]: This one should be cherry-picked.

File: provision-contest/ansible/roles/hosts/templates/hosts.j2 [hunk 2]
```diff
@@ -8,7 +8,9 @@ ff00::0 ip6-mcastprefix
 ff02::1 ip6-allnodes
 ff02::2 ip6-allrouters
 
+{% if DOMSERVER_IP is defined %}
```
Comment (nickygerritsen) [inline provision-contest/ansible/roles/hosts/templates/hosts.j2:11]: I think we need this to be cherry picked as well

File: provision-contest/ansible/roles/keepalived/templates/keepalived.conf.j2 [hunk 3]
```diff
@@ -1,6 +1,6 @@
 vrrp_instance lb_ipv4 {
 	state MASTER
-	interface {{ ansible_facts['default_ipv4']['interface'] }}
+	interface {{ KEEPALIVED_INTERFACE|default(ansible_facts['default_ipv4']['interface']) }}
```
Comment (nickygerritsen) [inline provision-contest/ansible/roles/keepalived/templates/keepalived.conf.j2:3]: This is based on what I did right? IMHO makes sense to cherry pick this if we don't have it in main yet

Comment (vmcj) [reply provision-contest/ansible/roles/keepalived/templates/keepalived.conf.j2:3]: Agree, will do the cherry-picking and rebasing tomorrow.

**Pull request closed without merging**